Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -1020,7 +1025,7 @@ void RenderObject::repaint() const
     issueRepaint();
 }
 
-void RenderObject::repaintRectangle(const LayoutRect& repaintRect, bool shouldClipToLayer) const
+void RenderObject::repaintRectangle(const LayoutRect& repaintRect, bool shouldClipToLayer, bool forceRepaint, bool shouldClipToContainer) const
```

Comment:
Can we have this take the enum values? The conversion between the bools and enum values is cumbersome.

Reply:
Yes I will make that change